app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -416,6 +425,7 @@ public static long insertLoyaltyCard(
         contentValues.put(LoyaltyCardDbIds.BALANCE_TYPE, balanceType != null ? balanceType.getCurrencyCode() : null);
         contentValues.put(LoyaltyCardDbIds.CARD_ID, cardId);
         contentValues.put(LoyaltyCardDbIds.BARCODE_ID, barcodeId);
+        contentValues.put(LoyaltyCardDbIds.USAGE_NUMBER, 0);
```

Comment:
This is below `ARCHIVE_STATUS` on top, please keep the order consistent everywhere.